core/domain/blog_services.py

```diff
@@ -598,35 +734,46 @@ def create_new_blog_post(author_id):
     return new_blog_post
 
 
-def get_published_blog_post_summaries(offset=0):
+def get_published_blog_post_summaries(
+    offset: int = 0
+) -> Optional[List[blog_domain.BlogPostSummary]]:
```

Comment:
Ditto, should we return empty list instead of `None`?

Reply:
This method is only used in `controllers` and there we are not handling the case of `None`. So i changed `None` to empty list.